frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationConstants.ts

```diff
@@ -0,0 +1,122 @@
+const SECONDS_PER_MINUTE = 60
+const MINUTES_PER_HOUR = 60
+const HOURS_PER_DAY = 24
+const DAYS_PER_WEEK = 7
+
+const SECONDS_PER_HOUR = SECONDS_PER_MINUTE * MINUTES_PER_HOUR
+const SECONDS_PER_DAY = SECONDS_PER_HOUR * HOURS_PER_DAY
+const SECONDS_PER_WEEK = SECONDS_PER_DAY * DAYS_PER_WEEK
+
+export const DEFAULT_FREQUENCY = `${SECONDS_PER_MINUTE * 30}` // 1800 seconds (30 minutes)
```

Comment:
Updated from 15 seconds, but wanted to double check that this is an appropriate default value?

Reply:
sounds good! we'll learn from our 1-day experiment if something larger is useful.